### ISSUE TITLE
Some shows are inexplicably "not available".

### ISSUE THREAD

**Issue by TartanSpartan** [ilyakooo0/podphoenix#20]

**Comment by TartanSpartan** [comment]
Per https://github.com/ilyakooo0/podphoenix/issues/20

**Comment by maciek134** [comment]
This is an issue with the iTunes API / whatever podcast authors put there. For example 
```
https://itunes.apple.com/search?term=Adeptus&media=podcast&entity=podcast
```
returns this `feedUrl` for the first podcast (the one I'm searching for):
```
https://anchor.fm/s/dc36d0bc/podcast/rss
```
But that returns 404 - possibly related to Spotify owning the domain now

**Comment by TartanSpartan** [comment]
Hmm, I will attempt to contact the show's technical team to address this.

**Comment by maciek134** [comment]
That's not a very good idea, we might as well be using a field that's no longer present in the Apple Podcasts UI

**Comment by maciek134** [comment]
@ilyakooo0 this can be used to look up episodes using the iTunes API too:
```
https://itunes.apple.com/lookup?id=<collectionId>&country=US&media=podcast&entity=podcastEpisode&limit=100
```
where `collectionId` is taken from the iTunes `/search` API. This includes `episodeUrl` which can be used to download the track. Somehow this works for the podcast I'm testing on, even though it points to a url on `anchor.fm` that redirects to cloudfront (I guess Spotify didn't break these)

**Comment by TartanSpartan** [comment]
So are you saying that using this URL to subscribe to a feed URL in Podphoenix:

https://itunes.apple.com/lookup?id=597919007&country=US&media=podcast&entity=podcastEpisode&limit=100

(with that ID corresponding to the show) should work to subscribe to the show? Because it doesn't for me.

**Comment by maciek134** [comment]
No, that's not what I said. I suggested a solution to fix this in the app, that URL is useless to you as a user.

**Comment by TartanSpartan** [comment]
Ok, the clarification is useful.

**Comment by maciek134** [comment]
Actuall this one was a completely different issue, the exact podcast you mention has a working feed URL but it was breaking due to a redirect - fixed in the PR linked above.

**Comment by TartanSpartan** [comment]
Heroic work, and thank you! But, do you know why only a handful of episodes (from the past week or so) are available for download?

**Comment by maciek134** [comment]
If you want the full list you have to go to the this podcast in the Podcasts submenu and refresh there, not sure if on purpose but refreshing in the main menu will only save a couple of latest episodes.
